api/bulk.py

```diff
@@ -0,0 +1,219 @@
+from __future__ import annotations
+
+import typing
+
+from django.db import models
+from rest_framework import response, serializers, status, viewsets
+from rest_framework.exceptions import ValidationError
+
+from kausal_common.const import IS_PATHS, IS_WATCH
+
+if typing.TYPE_CHECKING:
+    if IS_PATHS:
+        from paths.types import PathsAPIRequest as APIRequest
+    elif IS_WATCH:
+        from watch.types import WatchAPIRequest as APIRequest
```

Comment:
Wondering if there's a cleaner way to do this since it seems obvious it doesn't matter which APIRequest this is -> move the common base type to common if it's not there already